UnityProject/Assets/Scripts/Health/RespiratoryHealth/RespiratorySystem.cs

```diff
@@ -159,26 +162,43 @@ private float HandleBreathing(GasMix breathGasMix)
 		}
 		else
 		{
+			IsSuffocating = false;
 			oxygenUsed = breathGasMix.GetMoles(Gas.Oxygen);
 			bloodSystem.OxygenLevel += 30;
 		}
 
 		return oxygenUsed;
 	}
 
+	public enum PressureChecker {
```

Comment:
How come you're using this enum like a flag without the [Flag] attribute? I think it would be easier if you get rid of the bitwise operations and set this to the desired value directly instead, since it can only be one value at a time